### ISSUE TITLE
Location of installed modules

### ISSUE THREAD

**Issue by ifuchs** [holzschu/Carnets#85]
Where are the installed python modules stored?

**Comment by holzschu** [comment]
In `~/Library/lib/python-3.7/site-packages/`
or `$HOME/Library/lib/python-3.7/site-packages/`

**Comment by ifuchs** [comment]
Great, thanks. And can I view that Dir from a GUI app or do I use the usual
pythonic way of navigating the file system?

**Comment by holzschu** [comment]
It is not visible from the iOS GUI, unless maybe if you create a symbolic link first: `ln -s ~/Library/lib/python-3.7/site-packages/ ~/Documents/packages` 

Vim from inside a-Shell allows a sort-of user interface with `:Explore` (after `pickFolder` to select the Carnets directory). `NERDTree` works too, if you install it.

**Comment by ifuchs** [comment]
Ok, I installed Dplython and plotnine using pip however I don’t see them in
site packages. Is that where they should be? Also what is the easiest way
to get the nerdtree plugin? Is curl the preferred way?

**Comment by holzschu** [comment]
Yes, this is where they should be, unless the installation process failed. What does `pip show ` give you?

For nerdtree, I used WorkingCopy to clone the repository, followed by `pickFolder`, then `cp -r . ~/.vimrc/...`

**Comment by ifuchs** [comment]
I am probably off topic now but I have not used Working Copy but I
downloaded it and cloned the nerdtree repository. However if I use
pickFolder I don’t see a dir for Working Copy on my iPad. What needs to be
copied and where does it go (.vimrc or a plug-in directory or...?)

**Comment by holzschu** [comment]
WorkingCopy appears as a new file provider, along with "on my iPad" and "iCloud Drive". You might have to enable it by clicking on the "(...)" on the top of the leftmost column. 

Once you have pickFolder-ed inside the nerdtree repository, something like: `mkdir -p ~/Documents/.vim/pack/vendor/start/nerdtree` followed by `cp -r * ~/Documents/.vim/pack/vendor/start/nerdtree/*`.

**Comment by ifuchs** [comment]
Perfect. Thanks!
